R/macroEvaluation.R

```diff
@@ -21,33 +21,32 @@
 #' @author Mike K Smith \email{mstoolkit@@googlemail.com}
 #' @keywords datagen
 #' @examples
-#' 
+#'
 #'   # example of micro data with interim
-#'   microData <- read.csv( 
-#'     system.file( "Runit", "data", "macroEvaluation", "micro0001.csv" , package = "MSToolkit") ) 
+#'
 #'   mCode <- function(data) {
-#'     diffMeans <- data$MEAN[ data$DOSE == 100 & data$INTERIM == 0] - 
+#'     diffMeans <- data$MEAN[ data$DOSE == 100 & data$INTERIM == 0] -
 #'     data$MEAN[ data$DOSE == 0 & data$INTERIM == 0  ]
 #'     data.frame( SUCCESS = diffMeans > 10, NFINAL = sum(data$N) )
 #'   }
-#'   out <- macroEvaluation( microData, mCode)
+#'   out <- macroEvaluation( MSToolkit:::microData, mCode)
```

Comment:
Don't need the MSToolkit::: here as it points to the microData dataset above.
Should be:
```suggestion
#'   out <- macroEvaluation(microData, mCode)
```

Reply:
The original way to read the microData would generate an error when doing the R-CMD-check. So I assigned the microData as the internal data and we need to use ::: to call the internal data. But now I call the internal data by another way that is `get0("microData", envir = asNamespace("MSToolkit"))`.